<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyEditDialog</class>
 <widget class="QDialog" name="KeyEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="pathLabel">
           <property name="text">
            <string>path:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="messageLabel">
           <property name="text">
            <string>Press Key:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QKeySequenceEdit" name="keySequenceEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QCheckBox" name="repeatBox">
           <property name="text">
            <string>repeat</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="okButton">
           <property name="text">
            <string>&amp;OK</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cancelButton">
           <property name="text">
            <string>&amp;Cancel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QSlider" name="volumeSlider">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="volumeLabel">
           <property name="text">
            <string>Volume:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="volumeSpin">
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>100</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>volumeSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>volumeSpin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>volumeSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>volumeSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>375</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
